.travis.yml

```diff
@@ -2,9 +2,13 @@ language: ruby
 rvm:
   - 2.3.3
 before_install:
-  - cd site 
+  - cd site
   - 'if [ "$TRAVIS_PULL_REQUEST" = "false" ]; then echo $super_secret_password | gpg --passphrase-fd 0 service-acc.json.gpg; fi'
-script: 
+script:
+  - nvm install 7
+  - npm install
```

Comment:
I think both lines 8 and 9 probably makes more sense in the `before_install` step (after the first two commands already there)